src/main/kotlin/com/deviceinsight/helm/AbstractHelmMojo.kt

```diff
@@ -38,22 +39,33 @@ abstract class AbstractHelmMojo : AbstractMojo() {
 	@Parameter(property = "chartName", required = false)
 	private var chartName: String? = null
 
-	protected fun executeCmd(cmd: String, directory: File = target(),
-							 redirectOutput: ProcessBuilder.Redirect = ProcessBuilder.Redirect.PIPE) {
-		val proc = ProcessBuilder(cmd.split(" "))
+	protected fun executeCmd(
+		cmd: List<String>,
+		directory: File = target(),
+		redirectOutput: ProcessBuilder.Redirect = ProcessBuilder.Redirect.PIPE
+	) {
+		val proc = ProcessBuilder(cmd)
 			.directory(directory)
 			.redirectOutput(redirectOutput)
 			.redirectError(ProcessBuilder.Redirect.PIPE)
 			.start()
 
+		val stdoutPrinter = thread(name = "Stdout printer") {
+			proc.inputStream.bufferedReader().lines().forEach { log.debug("Output: $it") }
```

Comment:
Good catch. I didn't see that lint was logging to info. I guess thats the problem with having two methods that are doing roughly the same but slightly different.

I added an option to `executeCmd` for logging to `info` instead of `debug`. Also I hope it's ok if I still spawn a thread for logging stdout even when it is redirected to a file. This keeps the code a bit less noisy.

This will still work:
> If the standard output of the process has been redirected using ProcessBuilder.redirectOutput then this method will return a null input stream.